google/cloud/bigquery/v2/minimal/internal/job_options.cc

```diff
@@ -26,8 +27,24 @@ GOOGLE_CLOUD_CPP_INLINE_NAMESPACE_BEGIN
 
 namespace {
 auto constexpr kBackoffScaling = 2.0;
+std::size_t constexpr kJobDefaultConnectionPoolSize = 4;
+std::size_t constexpr kJobDefaultConnectionPoolSizeMax = 64;
 }  // namespace
 
+std::size_t DefaultConnectionPoolSize() {
```

Comment:
If the function isn't named `JobDefaultConnectionPoolSize()`, I'd probably remove the "Job" from the constant names too.